packages/discovery-provider/src/models/playlists/playlist.py

```diff
@@ -37,6 +37,8 @@ class Playlist(Base, RepresentableMixin):
     is_stream_gated = Column(Boolean, nullable=False, server_default=text("false"))
     stream_conditions = Column(JSONB(True))
     description = Column(String)
+    release_date = Column(DateTime)
```

Comment:
what's up with this one? Is it already in the db but missing from model?

Reply:
sry wdym? it is also in the migration.

Reply:
ohh. Can't read, mb 🤦